addons/web_editor/static/src/xml/editor.xml

```diff
@@ -12,35 +12,51 @@
                     data-bs-toggle="dropdown" data-bs-original-title="Text style" tabindex="-1" aria-expanded="false">
                     <span title="Text style">Normal</span>
                 </button>
-                <ul class="dropdown-menu">
-                    <li id="paragraph-dropdown-item">
-                        <a class="dropdown-item" href="#" id="paragraph" data-call="setTag" data-arg1="p">Normal</a>
+                <ul class="dropdown-menu" style="max-height:50vh;overflow-y:auto">
+                    <li id="display-1-dropdown-item">
+                        <a class="dropdown-item" href="#" id="display-1" data-call="setTag" data-arg1="h1,display-1">Header 1 Display 1</a>
```

Comment:
We should use the same label everywhere of course. Either "Header1 Display 2" or "Title 2" maybe "Header 1 Title 2" but a combination of Title and Display.

Reply:
I let @pierre-pde choose :+1: